rpcclient/wallet.go

```diff
@@ -764,14 +764,14 @@ func (r FutureAddMultisigAddressResult) Receive() (bchutil.Address, error) {
 		return nil, err
 	}
 
-	// Unmarshal result as a string.
-	var addr string
-	err = json.Unmarshal(res, &addr)
+	// Unmarshal result as a addmultisigaddress result object.
+	var addmultisigRes btcjson.AddMultisigAddressResult
+	err = json.Unmarshal(res, &addmultisigRes)
 	if err != nil {
 		return nil, err
 	}
 
-	return bchutil.DecodeAddress(addr, &chaincfg.MainNetParams)
+	return bchutil.DecodeAddress(addmultisigRes.Address, &chaincfg.MainNetParams)
```

Comment:
I am not familiar enough to be sure, but it looks like `DecodeAddress()` is setup to handle multisig addresses so I guess this whole code path should work now.

Reply:
Yep this should work as expected now. =)